andela/odin-ah-backend
# #160452465 - Setup PostgreSQL as DB Engine

#### What does this PR do?
Setup PostgreSQL as DB Engine
#### Description of Task to be completed?
- Installed and initialized Sequelize npm package with Postgres support
- Completely replaced mongoose with Sequelize
- Refactored user route functions into UserController
- Setup babel
#### How should this be manually tested?
* Before installing, ensure [PostgreSQL](https://www.postgresql.org/docs/9.4/static/tutorial-install.html) is installed and you've create a database.
* From the terminal, run the following commands to clone and install application dependencies. 
``` 
> git clone https://github.com/andela/odin-ah-backend
> cd odin-ah-backend
> git checkout chore-160452465-setup-postgresql-database
> npm install
``` 
* Rename `.env-sample` file on the root directory of the clone repo to `.env`. Uncomment `DB_URL` and supply the URL to your database.

```$xslt
> mv .env-sample .env
```

```
# DB_URI=postgres://<USERNAME>:<PASSWORD>@<HOST>:<PORT>/<database>
```
* To run application
 
 ```$xslt
 > npm start
 ```
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#160452465](<https://www.pivotaltracker.com/story/show/160452465>)
#### Screenshots (if appropriate)
- None
#### Questions:
